src/com/fediroryshchuk/homework7/brokesinglton/singletonpattern/Speaker.java

```diff
@@ -0,0 +1,18 @@
+package com.fediroryshchuk.homework7.brokesinglton.singletonpattern;
+
+public class Speaker extends SeniorGovernmentOfficials {
+    private static volatile Speaker instance;
+
```

Comment:
Add private constructor.

Reply:
If I make private constructor, I can’t extend class. Why do i need it?